assets/collapsible-content.css

```diff
@@ -85,8 +85,7 @@
 }
 
 .collapsible-row-layout .accordion summary, .collapsible-row-layout .accordion .accordion__content {
-  padding-left: 1.5rem;
-  padding-right: 1.5rem;
+  padding: 1.5rem;
```

Comment:
Thanks @melissaperreault 👍 

So I find that the spacing is a little much with this. Specifically the top padding 🤔  We could just add a padding bottom instead. 

Just wanted to bring it up and confirm we're good with this: [video](https://screenshot.click/16-56-09ifl-ojlw7.mp4)

Reply:
Thanks Ludo! 

It was the original intention to only add at the bottom but Lucas shared https://github.com/Shopify/dawn/issues/1597#issuecomment-1111448522 a good observation to optimize for hover and focus. I wasn't too distracted by the extra amount of padding at the top but would love another pair of eyes!

- Example [without padding-top](https://screenshot.click/16-16-xwpup-8f0nf.png)
- Example [with padding-top](https://screenshot.click/16-18-3dje5-6p9hq.png)

cc. @danielvan